lib/fabrik/BasePlatformManager.js

```diff
@@ -1,15 +1,12 @@
 'use strict';
 
-const CONST = require('../constants');
-
 class BasePlatformManager {
-  constructor(guid, context) {
-    this.guid = guid;
-    this.context = context;
+  constructor(platform) {
+    this.platform = platform;
   }
 
-  get platform() {
-    return this.context.platform;
+  getPlatformSpecificCatalog(catalog) {
```

Comment:
Platformmanager is to return things platformSpecifc :-), so may be can we strip this from the method name and just keep it `getCatalog`. Also can return `undefined` from BasePlatformManager as its implementation from specific manager is absent.

Reply:
:) changed it.